plugins/rxp/io/RxpPointcloud.cpp

```diff
@@ -79,39 +83,74 @@ RxpPointcloud::~RxpPointcloud()
 }
 
 
-point_count_t RxpPointcloud::read(PointViewPtr view, point_count_t count)
+bool RxpPointcloud::readOne(PointRef& point)
 {
-    point_count_t numRead = 0;
+    if (this->readSavedPoint(point)) {
```

Comment:
Would think `this` would be superfluous in this context.

Reply:
It's not technically necessary, but I like to use `this` to disambiguate the call from a free function call. Happy to remove if you'd like -- if so, maybe worth putting into the style guide as a recommended practice ("don't use `this` for method calls")?

Reply:
The style guide:

"Make your code look like existing code."

Of course, people are free to interpret this as they wish.

Reply:
Code updated via force-push per both comments.